topics:
  driving: cmd_vel 
  torso: {real: /torso_position_controller/command, sim: torso_effort_controller/command}
  head: /pan_tilt_trajectory_controller/command
  gripper: /gripper_controller/gripper_cmd

limits:
  driving: {scale_limit_linear: 0.07, scale_limit_angular: 0.3} # m/s
  torso: {lower: 0.05, upper: 0.4} # meters
  head: {pan_lower: -40, pan_upper: 40, tilt_lower: -30, tilt_upper: 50} # degrees
  gripper: {limit_lower: 0.01, limit_upper: 0.15, max_effort: 0.5}

start_pos: ninety_deg
